### PR TITLE
fix: compiling clr-angular without ivy so it could be compatible

### LINKED ISSUE #4826: Can't compile clr-angular with angular 10.0.5
## Describe the bug

```ts
ERROR in ./node_modules/@clr/angular/esm2015/utils/animations/expandable-animation/expandable-animation.js 33:8-43 "export 'ɵɵcomponentHostSyntheticListener' (imported as 'i0') was not found in '@angular/core'
ERROR in ./node_modules/@clr/angular/esm2015/utils/animations/expandable-animation/expandable-animation.js 35:8-39 "export 'ɵɵupdateSyntheticHostBinding' (imported as 'i0') was not found in '@angular/core'
ERROR in ./node_modules/@clr/angular/esm2015/utils/drag-and-drop/draggable-ghost.js 96:8-39 "export 'ɵɵupdateSyntheticHostBinding' (imported as 'i0') was not found in '@angular/core'
```

## How to reproduce

```bash
npm install @angular/cli@latest -g # make sure it is latest 10.0.5 ng new my-app
cd my-app
ng add @clr/angular@next # installing 4.0.0-rc.1
npm run start
```

## Expected behavior

Run application without erros

## Versions

**App**

- Angular: 10.0.5
- Node: 12.0.x
- Clarity: 4.0.0-rc.1

### PR DESCRIPTION
**Pull request by bdryanovski**
* Fixing version to ^@angular/<package>@10.0.0 as the minimum required version
* Updating missing properties inside the `package.json`
* Building the `clr-angular` package with `--prod`
* Updating yarn.lock file

- [x] Bugfix
- [x] Refactoring (no functional changes, no api changes)
- [x] Build related changes

## What is the current behavior?

```
ERROR in ./node_modules/@clr/angular/esm2015/utils/animations/expandable-animation/expandable-animation.js 33:8-43 "export 'ɵɵcomponentHostSyntheticListener' (imported as 'i0') was not found in '@angular/core'
ERROR in ./node_modules/@clr/angular/esm2015/utils/animations/expandable-animation/expandable-animation.js 35:8-39 "export 'ɵɵupdateSyntheticHostBinding' (imported as 'i0') was not found in '@angular/core'
ERROR in ./node_modules/@clr/angular/esm2015/utils/drag-and-drop/draggable-ghost.js 96:8-39 "export 'ɵɵupdateSyntheticHostBinding' (imported as 'i0') was not found in '@angular/core'
```

Issue Number: #4826

## What is the new behavior?

```
Compiling @clr/angular : es2015 as esm2015

chunk {main} main.js, main.js.map (main) 10.2 kB [initial] [rendered]
chunk {polyfills} polyfills.js, polyfills.js.map (polyfills) 141 kB [initial] [rendered]
chunk {runtime} runtime.js, runtime.js.map (runtime) 6.15 kB [entry] [rendered]
chunk {styles} styles.js, styles.js.map (styles) 12.4 kB [initial] [rendered]
chunk {vendor} vendor.js, vendor.js.map (vendor) 4.67 MB [initial] [rendered]
Date: 2020-07-28T22:28:57.162Z - Hash: 28ce26e1fe39e35cb4e4 - Time: 9703ms
** Angular Live Development Server is listening on localhost:4200, open your browser on http://localhost:4200/ **
: Compiled successfully.

Date: 2020-07-28T22:28:57.825Z - Hash: 28ce26e1fe39e35cb4e4
5 unchanged chunks

Time: 366ms
: Compiled successfully.
```

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Other information

Close #4826

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -22,6 +22,14 @@
     "@angular-devkit/core" "10.0.0-rc.3"
```
Comment (gnomeontherun) [inline yarn.lock:22]: I can see some references to RC3, is this right? I'm confused how we have different installed versions.

File: packages/angular/npm.json [hunk 2]
```diff
@@ -2,10 +2,11 @@
   "name": "@clr/angular",
   "version": "@VERSION",
   "description": "Angular components for Clarity",
-  "es2015": "esm2015/clr-angular.js",
   "homepage": "https://clarity.design",
+  "es2015": "fesm2015/clr-angular.js",
```
Comment (gnomeontherun) [inline packages/angular/npm.json:6]: Why did you make these changes?

File: packages/angular/package.json [hunk 3]
```diff
@@ -56,8 +56,8 @@
   "devDependencies": {
     "@angular-devkit/build-angular": "0.1000.0-rc.3",
```
Comment (gnomeontherun) [inline packages/angular/package.json:57]: We should bump this too.

**Comment by bdryanovski** [comment]
fixed into antother PR #4830

**Pull request closed without merging**